### PR TITLE
Update 2021-05-02-os-command-injection-in-rdoc.md

### PR DESCRIPTION
**Pull request by unak**
Mention about Ruby 3.0.2, Ruby 2.7.4 and Ruby 2.6.8

### CODE REVIEW

File: en/news/_posts/2021-05-02-os-command-injection-in-rdoc.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Ruby users whose version of RDoc is affected by this issue should update to the
 ## Affected Versions
 
 * All releases of RDoc from 3.11 to 6.3.0
+* Ruby 3.0.2 (bundles RDoc 6.3.1), Ruby 2.7.4 (bundles RDoc 6.2.1.1) and Ruby 2.6.8 (bundles RDoc 6.1.2.1) include fixes and not affected
```
Comment (nagachika) [inline en/news/_posts/2021-05-02-os-command-injection-in-rdoc.md:27]:
```suggestion
* Ruby 3.0.2 (bundles RDoc 6.3.1), Ruby 2.7.4 (bundles RDoc 6.2.1.1) and Ruby 2.6.8 (bundles RDoc 6.1.2.1) include the fixes and not affected
```

Comment (Nakilon) [reply en/news/_posts/2021-05-02-os-command-injection-in-rdoc.md:27]: also probably

> and **are** not affected

**Comment by yous** [comment]
Any updates here?

**Review by sorah — APPROVED** [review]
@unak @nagachika feel free to merge once you update the timestamp in the History section

**Pull request closed without merging**